zmon_aws_agent/aws.py

```diff
@@ -192,6 +192,15 @@ def get_running_apps(region):
                 'infrastructure_account': 'aws:{}'.format(owner),
             }
 
+            i["block_devices"] = {}
+            for device in i.get("BlockDeviceMappings", []):
+                if "Ebs" in device:
+                    i["block_devices"][device["DeviceName"]] = {
+                        "volume_id": device["Ebs"]["VolumeId"],
+                        "volume_type": "ebs",
```

Comment:
is there a device['Ebs']['VolumeType']? We need it in order to filter only gp2 ones.